libsolidity/interface/StandardCompiler.cpp

```diff
@@ -1503,6 +1527,12 @@ Json StandardCompiler::compileSolidity(StandardCompiler::InputsAndSettings _inpu
 				creationJSON["linkReferences"] = formatLinkReferences(compilerStack.object(contractName).linkReferences);
 			if (evmCreationArtifactRequested("generatedSources"))
 				creationJSON["generatedSources"] = compilerStack.generatedSources(contractName, /* _runtime */ false);
+			if (evmCreationArtifactRequested("subAssemblyOffsets"))
+			{
+				Json ret = Json::object();
+				ret["subs"] = formatSubAssemblyOffsets(compilerStack.object(contractName).subAssemblyData);
+				creationJSON["subAssemblyOffsets"] = std::move(ret);
+			}
```

Comment:
Just FYI, with nlohmann-json you can create JSON objects in place:
```suggestion
				creationJSON["subAssemblyOffsets"] = {
					{"subs", formatSubAssemblyOffsets(compilerStack.object(contractName).subAssemblyData)}
				};
```

We should generally use that style whenever possible, because the intermediate variables does not add anything of value, especially one with a horrible name like `ret`. And the reversed order always makes things harder to follow.